src/types.jl

```diff
@@ -221,41 +221,51 @@ mutable struct ISpatialRef <: AbstractSpatialRef
     end
 end
 
-function _infergeomtype(ptr::GDAL.OGRGeometryH = C_NULL)::OGRwkbGeometryType
-    return if ptr != C_NULL
-        convert(OGRwkbGeometryType, GDAL.ogr_g_getgeometrytype(ptr))
-    else
+_infergeomtype(ptr::GDAL.OGRGeometryH) = wkbUnknown
```

Comment:
Should this line be removed? It doesn't seem to be covered by unit tests.

Reply:
removed